src/components/Dropdown/Dropdown.ts

```diff
@@ -83,6 +83,9 @@ namespace fabric {
         if (option.disabled) {
           newItem.classList.add(IS_DISABLED_CLASS);
         }
+		if(option.selected) {
```

Comment:
Indentation/spacing is a little off here. Could you make this match the formatting used on lines 83 - 85?